src/backend/migrations/010-delete-user.sql

```diff
@@ -0,0 +1,47 @@
+-- ## Migration to:
+-- * Enable user data deletion
+
+-- Start a transaction
+BEGIN;
+
+-- Add column 'username' to 'task_events' table
+DO $$
+BEGIN
+    IF NOT EXISTS (
+        SELECT 1
+        FROM information_schema.columns
+        WHERE table_name='task_events' AND column_name='username'
+    ) THEN
+        ALTER TABLE public.task_events
+        ADD COLUMN username VARCHAR(255);
```

Comment:
No need to specify max 255 chars - it has no efficiency gain in Postgres. Just VARCHAR is good 👍